libvirt/README.md

```diff
@@ -0,0 +1,54 @@
+# Machine Stats for Virtual environments (Virt-Stats)
+
+A simple and effective way to gather guest VM statistics (hostname, IP addresses) from a [libvirt](https://libvirt.org/)-based environment as a first layer of a [Tidal Migrations discovery process](https://guides.tidalmg.com/).
+
+## Prerequisutes
+
+* [Python 3+](https://python.org/)
+* [libvirt](https://libvirt.org/) >=3.0.0 installed on both operator machine (i.e where you run Virt-Stats) and on remote machine (i.e where you run your virtual environmnent and guest VMs)
+* [QEMU guest agent](https://wiki.qemu.org/Features/GuestAgent) installed and running on guest VMs _(optional)_.
+
+## Installation
+
+Install locally in a Python 3 environment:
+
+```
+python3 -m pip install virt-stats
+```
+
+## Data captured
+
+As of now Virt-Stats captures the following metrics:
+
+* Hostname
+* IP Addresses
+
+## Usage
+
+```
+virt-stats --connection URI
```

Comment:
maybe we could add a note here for how the connection string looks like, WDYT?

Reply:
Added link to the details page in `--help` message in https://github.com/tidalmigrations/machine_stats/pull/70/commits/18bede59cc8b834401acd5aa46e10fcb3a9385a2